### DISCUSSION TITLE
Having trouble installing this

### DISCUSSION THREAD

**Discussion by Danielson191** [Arnuh/RuneLiteHijack#12] [General]
I can’t seem to find the config.json file. I’ve found .runelite but doesn’t seem to be config in there

**Comment by Arnuh** [comment]
Wrong directory, you need your launcher install directory.

Can search runelite in your start menu, right click & open file location. Right click the shortcut and open file location

**Comment by Danielson191** [comment]
Hi I’ve managed to install it thanks but I’m also struggling to find how to get a repository in raw GitHub user content form sorry I’m not very computer literate

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Danielson191** [comment]
This isn’t showing any plugins when I try to install it

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Danielson191** [comment]
Is there anyway you can send me your plugins to copy from ?

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
